index.d.ts

```diff
@@ -1,6 +1,6 @@
 declare module "react-native-matomo-sdk" {
   namespace MatomoSDK {
-    export function initialize(apiUrl: string, siteId: number): Promise<void>;
+    export function initialize(apiUrl: string, siteId: number, contentBase: string): Promise<void>;
```

Comment:
I don't think the `contentBase` should be a required parameter, as this would require everyone using the library to update their code on the next release. Instead it would be better if it was an optional parameter, which would default to the current behaviourl